src/components/chip/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { FC, PropsWithChildren } from 'react';
+import { Box, Flex, FlexProps, Text, useRecipe } from '@chakra-ui/react';
+
+import { chipRecipe } from 'src/themes/shared/recipes/chip';
+import { CheckmarkIcon } from 'src/components/icons/CheckmarkIcon';
+
+export type ChipProps = {
+  variant?: 'default' | 'selected' | 'disabled';
```

Comment:
Please use `RecipeVariantProps` here so we don't need to keep the variants in sync with recipe manually